aptos-move/block-executor/src/view.rs

```diff
@@ -1065,14 +1217,18 @@ impl<'a, T: Transaction, S: TStateView<Key = T::Key>, X: Executable> TResourceGr
         resource_tag: &Self::ResourceTag,
         _maybe_layout: Option<&Self::Layout>,
     ) -> anyhow::Result<Option<Bytes>> {
-        // TODO[agg_v2]: support layouts.
```

Comment:
When we read from a group, we are reading a member at (key, tag, layout). If it is not in latest view, we initialize the whole group, but we know the layout only for the member. So we have to initialize the members to LayoutInfo::Uninitialized, and the particular member that we read as LayoutInfo::Some(...) if we have Some(layout). On the read when the group is cached in the view, it can be that layout is not initialized, so we have to change it from LayoutInfo::Uninitialized to LayoutInfo::Some() or LayoutInfo::None depending on maybe_layout.